src/components/Validator.vue

```diff
@@ -62,6 +63,13 @@
 import { Validator } from '@xsor/tlsv';
 import Disclaimer from '@/components/Disclaimer';
 
+// Those mappings must be the same as defined in the '@xsor/tlsv' module.
+const FileFormat = 1
+const ChannelCount = 2
+const FseqType = 3
+const Duration = 4
+const Memory = 5
```

Comment:
These can be imported from the validator package.

Reply:
Tried around a while trying to import these - just nothing seamed to work. It would definitely be a better approach to import these...